server/sonar-server/src/main/java/org/sonar/server/permission/UserPermissionChanger.java

```diff
@@ -70,7 +69,7 @@ private boolean shouldSkipChange(DbSession dbSession, UserPermissionChange chang
 
   private void checkOtherAdminUsersExist(DbSession session, PermissionChange change) {
     if (GlobalPermissions.SYSTEM_ADMIN.equals(change.getPermission()) &&
-      !change.getProjectRef().isPresent() &&
+      !change.getProjectId().isPresent() &&
       dbClient.roleDao().countUserPermissions(session, change.getPermission(), null) <= 1) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "countUserPermissions"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)